{"name": "密码错误用例", "status": "passed", "description": "测试用例描述：此条用例期待结果：failed_password", "parameters": [{"name": "case_title", "value": "'密码错误用例'"}, {"name": "username", "value": "'zhangsan'"}, {"name": "password", "value": "'passwd'"}, {"name": "expect_result", "value": "'failed_password'"}], "start": 1692193859648, "stop": 1692193859648, "uuid": "ea8da9a3-0c98-4072-a631-217a06e93907", "historyId": "5025d9744a829d60718a69baf6abdb4d", "testCaseId": "bf013d8ec2d7a7fb4b3fad4fe3025af7", "fullName": "testcase.test_pytest9#test_login", "labels": [{"name": "story", "value": "用户故事-登录测试"}, {"name": "epic", "value": "商城项目"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_pytest9"}, {"name": "host", "value": "LAPTOP-DDAF8AKP"}, {"name": "thread", "value": "2612-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_pytest9"}]}